packages/queued-request-controller/CHANGELOG.md

```diff
@@ -6,4 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-[Unreleased]: https://github.com/MetaMask/core/
+## [0.1.0]
+### Added
+- Initial release
+- Add `QueuedRequestController` class and `QueuedRequestMiddleware`.
```

Comment:
can probably just leave off and only have the `Initial release` line item

Reply:
done https://github.com/MetaMask/core/pull/1904/commits/c401a88c35d4ec47934bd48847d74e75b7643aea